dist/tcti-tabrmd.pc.in

```diff
@@ -1,7 +1,7 @@
 Name: tcti-tabrmd
 Description: TCTI library for communicating with the TPM2 access broker / resource manager daemon (tabrmd).
-URL: https://github.com/01org/??
+URL: https://github.com/01org/tpm2-abrmd
 Version: @VERSION@
-Requires: sapi
+Requires: sapi glib-2.0 gio-2.0
```

Comment:
If we omit the gio dependency in the 'pc' file here then the build breaks since gio headers / libs won't be known to the Makefile.